nautobot/dcim/templates/dcim/powerfeed.html

```diff
@@ -109,16 +109,16 @@ <h1>{% block title %}{{ object }}{% endblock %}</h1>
                 <tr>
                     <td>Utilization (Allocated)</td>
                     {% with utilization=object.connected_endpoint.get_power_draw %}
-                        {% if utilization %}
-                            <td>
-                                {{ utilization.allocated }}VA / {{ object.available_power }}VA
-                                {% if object.available_power > 0 %}
-                                    {% utilization_graph utilization.allocated|percentage:object.available_power %}
-                                {% endif %}
-                            </td>
-                        {% else %}
-                            <td class="text-muted">N/A</td>
+                    {% if utilization %}
```

Comment:
Please follow the existing indentation style of this file (`{% if utilization %}` should be indented four spaces under the `{% with utilization=...` to improve readability of the logic).